code/modules/clothing/gloves/miscellaneous.dm

```diff
@@ -166,7 +166,7 @@
 	var/mob/living/M = loc
 
 	if(M.a_intent in accepted_intents)
-		if(istype(M.mind.martial_art, /datum/martial_art/cqc))
+		if(M.mind.martial_art, /datum/martial_art) || HAS_TRAIT(M, TRAIT_HULK))
```

Comment:
This is what I meant. Checking if the `martial_art` var is something other than null, which implies they currently have a martial art.
```suggestion
		if(M.mind.martial_art) || HAS_TRAIT(M, TRAIT_HULK))
```